polyjuice/dicom_image.py

```diff
@@ -1,14 +1,24 @@
 import dicom
-
+import os
```

Comment:
not needed but I prefer standard library imports to be ahead of the 3rd party ones and with a line break between them